Lambda-School-Labs/mydish-fe
# Fix/cookbook not updating

Previously, if you created, updated, or deleted a recipe, that change wouldn't be reflected in the CookBook. This PR fixes that.

**To test:**
1. Create a new recipe.
2. Check the CookBook tab. Is your recipe there?
3. Edit the recipe. Change the title, the image, and the tags. Save.
4. Check the CookBook again. Are all your changes reflected there?
5. Delete the recipe.
6. Check the CookBook one last time. Has the recipe been removed?
